tests/safety/test_toyota.py

```diff
@@ -358,6 +362,21 @@ def test_lta_2_steer_cmd(self):
       should_tx = not req and not req2 and angle == 0
       self.assertEqual(should_tx, self._tx(self._lta_2_msg(req, req2, angle)), f"{req=} {req2=} {angle=}")
 
+  def _accel_2_msg(self, accel, cancel_req=0):
```

Comment:
Update function name to override the base function in common test. This should clear up the error in the mutation testing.
```suggestion
  def _accel_msg_2(self, accel, cancel_req=0):
```